fw-child/template/query/learn-item.php

```diff
@@ -1,3 +1,19 @@
+<?php
+// Initialize current language.
+$current_lang = 'en';
+
+if ( isset( $item['lang'] ) && in_array( $item['lang'], array( 'en', 'fr' ), true ) ) {
+	$current_lang = $item['lang'];
+}
+
+$native_excerpt = apply_filters ( 'the_content', custom_excerpt ( 20, $item['id'] ) );
+
+$excerpts = array (
+    'en' => $native_excerpt ?: get_field( 'excerpt', $item['id'] ), // Overrides native excerpt with custom excerpt if present
+    'fr' => get_field( 'excerpt_fr', $item['id'] ) ?: $native_excerpt // Defaults back to native excerpt if no custom FR not present
```

Comment:
It is discouraged to use the ternary operator (?:) because it might be a little confusing. Please fix that when exist on this MR. Also, adding comma (,) at the end of line is recommended by WordPress Coding Standards.